src/Pages/FarmTImeLine.js

```diff
@@ -79,118 +106,144 @@ function FarmTimeline() {
-              </span>
-            </ListItem>
-            <ListItem key="1">
-              <FormControl className={classes.formControl}>
-                <InputLabel id="locality-label">Locality</InputLabel>
-                <Select
-                  id="locality"
-                  name="locality"
-                  value={locality}
-                  onChange={handleChange}
-                >
-                  {locationsDataFromServer.map((locationDataFromServer) => {
-                    return (
-                      <MenuItem
-                        key={locationDataFromServer.code}
-                        value={locationDataFromServer.code}
-                      >
-                        {locationDataFromServer.displayStr}
-                      </MenuItem>
-                    );
-                  })}
-                </Select>
-              </FormControl>
-            </ListItem>
-
-            <ListItem key="2">
-              <FormControl className={classes.formControl}>
-                <InputLabel id="fId-label">Field ID</InputLabel>
-                <Select
-                  id="fieldId"
-                  name="fieldId"
-                  value={fieldId}
-                  onChange={handleChange}
-                >
-                  {fields.map((field) => {
-                    return (
-                      <MenuItem key={field.identifier} value={field.identifier}>
-                        {field.identifier}
-                      </MenuItem>
-                    );
-                  })}
-                </Select>
-              </FormControl>
-            </ListItem>
-
-            <ListItem>
-              <MuiPickersUtilsProvider utils={DateFnsUtils}>
-                <Grid container justify="space-around">
-                  <KeyboardDatePicker
-                    disableToolbar
-                    variant="inline"
-                    format="MM/dd/yyyy"
-                    margin="normal"
-                    id="start-date"
-                    label="From"
-                    value={selectedDate}
-                    onChange={handleStartDateChange}
-                    KeyboardButtonProps={{
-                      "aria-label": "change date",
-                    }}
-                  />
-                </Grid>
-              </MuiPickersUtilsProvider>
-            </ListItem>
-
-            <ListItem>
-              <MuiPickersUtilsProvider utils={DateFnsUtils}>
-                <Grid container justify="space-around">
-                  <KeyboardDatePicker
-                    disableToolbar
-                    variant="inline"
-                    format="MM/dd/yyyy"
-                    margin="normal"
-                    id="en-date"
-                    label="To"
-                    value={selectedEndDate}
-                    onChange={handleEndateChange}
-                    KeyboardButtonProps={{
-                      "aria-label": "change date",
+    <div>
+      {!showTimeLine ?
```

Comment:
Why is showing the form conditional? We need to sow the form always, the timeline should appear below the form fields. Please update.

Reply:
done